text_extensions_for_pandas/array/test_tensor.py

```diff
@@ -260,6 +260,12 @@ def test_sum(self):
         sum_some = df["s"][[True, False, True]].sum()
         npt.assert_array_equal(sum_some.to_numpy(), [6, 8])
 
+    def test_factorize(self):
+        x = np.array([[1, 2], [3, 4], [5, 6], [3, 4]])
+        s = TensorArray(x)
+        with self.assertRaises(NotImplementedError):
+            indices, values = s.factorize()
+
```

Comment:
Can you add a test of `__equals__` here too?

Reply:
I have a test that does all the binary operators, so it was being tested there